Dockerfile_amd

```diff
@@ -75,8 +86,8 @@ RUN chmod +x ~/mambaforge.sh && \
     mamba init && \
     rm ~/mambaforge.sh
 
-# Install PyTorch 2.2 RC compiled against RoCm 5.7, as VLLM can not be compiled with RoCm 5.6.
-RUN pip install torch --index-url https://download.pytorch.org/whl/test/rocm5.7/
+# Install PyTorch 2.3 RC compiled against RoCm 6.0
+RUN pip install torch numpy --index-url https://download.pytorch.org/whl/test/rocm6.0
```

Comment:
Let's update to torch nightly once https://github.com/pytorch/pytorch/pull/124362 is merged and wheels are published